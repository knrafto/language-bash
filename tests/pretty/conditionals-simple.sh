if true; then true; fi
if true; then true; elif false; then false; fi
if true; then true; else false; fi
if true; then true; elif false; then false; else false; fi
case x in x) true;; esac
select x; do true; done
select x in y z; do true; done
