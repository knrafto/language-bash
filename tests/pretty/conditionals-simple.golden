if true; then
    true;
fi;
if true; then
    true;
else
    if false; then
        false;
    fi;
fi;
if true; then
    true;
else
    false;
fi;
if true; then
    true;
else
    if false; then
        false;
    else
        false;
    fi;
fi;
case x in
    x)
        true;
        ;;
esac;
select x; do
    true;
done;
select x in y z; do
    true;
done;
